src/main/resources/db/migration/V1_3__create_tables.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE VURDERING
+(
+    id                 SERIAL PRIMARY KEY,
+    uuid               CHAR(36)    NOT NULL UNIQUE,
+    personident        VARCHAR(11) NOT NULL,
+    created_at         timestamptz NOT NULL,
+    updated_at         timestamptz NOT NULL,
+    created_by         VARCHAR(7)  NOT NULL,
+    status             VARCHAR(20) NOT NULL,
+    begrunnelse        TEXT,
+);
+
+CREATE INDEX IX_VURDERING_PERSONIDENT on VURDERING (personident);
+
+CREATE TABLE VARSEL
+(
+    id                          SERIAL PRIMARY KEY,
+    uuid                        CHAR(36)    NOT NULL UNIQUE,
+    created_at                  timestamptz NOT NULL,
+    updated_at                  timestamptz NOT NULL,
+    vurdering_id                INTEGER     NOT NULL UNIQUE REFERENCES VURDERING (id) ON DELETE CASCADE,
+    document                    JSONB       NOT NULL DEFAULT '[]'::jsonb,
+    journalpost_id              VARCHAR(20)
```

Comment:
Denne vil settes ved journalføring. 
Mulig vi trenger `svarfrist` her også, hvis ikke det er godt nok å alltid utlede den fra created_at. En fordel med å lagre den er at vi kan vise den i frontend (som gjør med forhåndsvarsel der) uten å måtte duplisere logikken for å regne den ut.
Trenger antagelig noe `published*`-timestampz her også, men det kan vi utvide med senere.